### PR TITLE
Home designs desktop top nav changes

### PR DESCRIPTION
**Pull request by adamgall**
Closes https://github.com/decentdao/decent-interface/issues/1905

Implements everything in the attached issue except for `max-width: 400px` on the search input. That causes all kinds of issues, including:
- ethereum address doesn't fit
- struggling to _not_ apply that max-width on the search input in full-screen modal, _and_ make the search result popup have the correct width

### CODE REVIEW

File: src/i18n/locales/en/dashboard.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "titleFavorites": "My Safes",
-  "emptyFavorites": "You have not added any Safes yet.",
+  "emptyFavorites": "You haven't added any Safes yet.",
+  "emptyFavorites2": "Create or find a Safe above.",
```
Comment (DarksightKellar) [inline src/i18n/locales/en/dashboard.json:4]: `emptyFavoritesSubtext` or `emptyFavoritesSubtitle`?

**Review by DarksightKellar — APPROVED** [review]
Name nitpick, looks good to me otherwise

File: src/i18n/locales/en/dashboard.json (hunk 1 above)
Comment (adamgall) [reply src/i18n/locales/en/dashboard.json:4]: Haha good call

Comment (adamgall) [reply src/i18n/locales/en/dashboard.json]: Fixed and pushed

**Pull request merged**